docs/GraphQL-API/query/attendee.md

```diff
@@ -6,31 +6,31 @@ Attendee object has connections with `RootQuery`.
 
 ```gql
 query GET_ATTENDEES($where: EspressoRootQueryAttendeesConnectionWhereArgs) {
-	espressoAttendees(where: $where) {
-		edges {
-			node {
-				id
-				dbId
-				avatar
-				firstName
-				lastName
-				bio
-				shortBio
-				email
-				phone
-				address
-				address2
-				city
-				zip
-				country {
-					name
-				}
-				state {
-					name
-				}
-			}
-		}
-	}
+    espressoAttendees(where: $where) {
+        edges {
```

Comment:
can you plz fix your formatting and put it back to tabs instead of spaces for indents

Reply:
It was prettier messing up. Had to update `.editorconfig` to fix it.